dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/QuayImageRegistry.java

```diff
@@ -221,4 +217,42 @@ private void updateContainersWithBuildInfo(SimpleDateFormat formatter, Map<Strin
         container.setRegistry(Registry.QUAY_IO);
         container.setGitUrl(gitURL);
     }
+
+    // private void checkTriggers(Container container) {
```

Comment:
Don't check in unused code as comments. 
Move this out into a library class or similar if you think it will have some use down the road.